tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial
+title: "International Flutter Apps"
+
+permalink: /tutorials/international/
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* How to track the device's locale (the user's preferred language).
+* How to manage locale-specific app values.
```

Comment:
Is there another word for "values" ? Not sure... but seems really vague...

Reply:
I was using "resources" but that's vague and pretentious, so I switched to just "values". 

Usually the values are just strings, but they don't have to be.